src/CattyTests/ViewController/Scene/ScenePresenterViewControllerSpy.swift

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Could you please move this file to _CattyTests/Mocks_? Mocks is used more as a generic term for test doubles here, so also Spies should to in there so that they can be easily found and reused in other tests.